src/control/numTrees/Collision.cpp

```diff
@@ -72,7 +72,7 @@ void Collision::setGoalCollision(const Vector2 &collisionPos, double distance) {
 
 const Vector2 Collision::collisionPosition() const {
     if (collisionRobot->id != - 1) return collisionRobot->pos;
-    else if (collisionBall->visible) return collisionBall->pos;
+    else if (collisionBall->getVisible()) return collisionBall->getPos();
```

Comment:
getVisible now gets the value of 'visiblyByAnyCamera' right? maybe keep the names consistent?

Reply:
getVisibleByAnyCamera() is a very long method name. I decided to make the function names of the getters and setters short and the actual variable names long, to keep the right balance between the size of the code and the understandability of the code.